### DISCUSSION TITLE
mt终于保号了 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#133] [Announcements]
https://blog.goalonez.site/blog/mt%E7%BB%88%E4%BA%8E%E4%BF%9D%E5%8F%B7%E4%BA%86.html 

 Goalonez Blog

**Comment by andrew-hongguo** [comment]
can you invite me to MT? thanks

**Comment by asteedma** [comment]
can you invite me to MT?! thanks

**Comment by maidou-00** [comment]
求问博主，没有盒子的话是如何刷到这种上传量的。。。大量的辅种？有什么技巧嘛

**Comment by Goalonez** [comment]
> @maidou-00
> 求问博主，没有盒子的话是如何刷到这种上传量的。。。大量的辅种？有什么技巧嘛

我也没有用盒子，就是正常家宽。
我是保种了一些豆瓣前100的热门资源，每天基本会都有一些流量在跑。放几个月也就差不多了。
辅种的话只辅你要玩的站就好了，不然吃了上传你后面可能也不太会用，就浪费了。
其实pt也就是刚开始玩上传量会不够，放一段时间后根本用不完😂

**Comment by maidou-00** [comment]
> @Goalonez
> > @maidou-00
> > 求问博主，没有盒子的话是如何刷到这种上传量的。。。大量的辅种？有什么技巧嘛
> 
> 我也没有用盒子，就是正常家宽。
> 我是保种了一些豆瓣前100的热门资源，每天基本会都有一些流量在跑。放几个月也就差不多了。
> 辅种的话只辅你要玩的站就好了，不然吃了上传你后面可能也不太会用，就浪费了。
> 其实pt也就是刚开始玩上传量会不够，放一段时间后根本用不完😂

谢谢博主指点，原本以为你不会看回复的哈哈。另外还想请教一下，我在博主另一个帖子里看了下PT下载和MoviePilot整理思路，里面还有些地方不太明白，想请教下：

我目前是用qBittorrent下PT，有个2T的SSD专门用来刷上传，通常下载完毕后1-2天后就没上传量了，这时候我想转移到仓库盘里面保种/刷魔力。
转移的时候，我想着把“A-要看的电影”和“B-完全用来保种的电影“都放到同一个文件夹，但是这样，影视库就显得很杂乱；因此我希望把B专门放到一个“保种文件夹”里面专门保种，但是这样，A这部分就没法保种了，其实量也不小。因此想和博主请教一下，有什么办法可以把A也保种吗？

**Comment by Goalonez** [comment]
> @maidou-00
> > @Goalonez
> > > @maidou-00
> > > 求问博主，没有盒子的话是如何刷到这种上传量的。。。大量的辅种？有什么技巧嘛
> > 
> > 我也没有用盒子，就是正常家宽。
> > 我是保种了一些豆瓣前100的热门资源，每天基本会都有一些流量在跑。放几个月也就差不多了。
> > 辅种的话只辅你要玩的站就好了，不然吃了上传你后面可能也不太会用，就浪费了。
> > 其实pt也就是刚开始玩上传量会不够，放一段时间后根本用不完😂
> 
> 谢谢博主指点，原本以为你不会看回复的哈哈。另外还想请教一下，我在博主另一个帖子里看了下PT下载和MoviePilot整理思路，里面还有些地方不太明白，想请教下：
> 
> 我目前是用qBittorrent下PT，有个2T的SSD专门用来刷上传，通常下载完毕后1-2天后就没上传量了，这时候我想转移到仓库盘里面保种/刷魔力。
> 转移的时候，我想着把“A-要看的电影”和“B-完全用来保种的电影“都放到同一个文件夹，但是这样，影视库就显得很杂乱；因此我希望把B专门放到一个“保种文件夹”里面专门保种，但是这样，A这部分就没法保种了，其实量也不小。因此想和博主请教一下，有什么办法可以把A也保种吗？

影视库里的应该是MoviePilot刮削硬链接后的路径，保种的则是你原先下载的路径，你下载的路径不动，硬链接的无论怎么改都没关系，应该都不影响。当然前提是没有跨盘操作。

**Comment by maidou-00** [comment]
> > [@maidou-00](https://github.com/maidou-00)
> > > [@Goalonez](https://github.com/Goalonez)
> > > > [@maidou-00](https://github.com/maidou-00)
> > > > 求问博主，没有盒子的话是如何刷到这种上传量的。。。大量的辅种？有什么技巧嘛
> > > 
> > > 
> > > 我也没有用盒子，就是正常家宽。
> > > 我是保种了一些豆瓣前100的热门资源，每天基本会都有一些流量在跑。放几个月也就差不多了。
> > > 辅种的话只辅你要玩的站就好了，不然吃了上传你后面可能也不太会用，就浪费了。
> > > 其实pt也就是刚开始玩上传量会不够，放一段时间后根本用不完😂
> > 
> > 
> > 谢谢博主指点，原本以为你不会看回复的哈哈。另外还想请教一下，我在博主另一个帖子里看了下PT下载和MoviePilot整理思路，里面还有些地方不太明白，想请教下：
> > 我目前是用qBittorrent下PT，有个2T的SSD专门用来刷上传，通常下载完毕后1-2天后就没上传量了，这时候我想转移到仓库盘里面保种/刷魔力。
> > 转移的时候，我想着把“A-要看的电影”和“B-完全用来保种的电影“都放到同一个文件夹，但是这样，影视库就显得很杂乱；因此我希望把B专门放到一个“保种文件夹”里面专门保种，但是这样，A这部分就没法保种了，其实量也不小。因此想和博主请教一下，有什么办法可以把A也保种吗？
> 
> 影视库里的应该是MoviePilot刮削硬链接后的路径，保种的则是你原先下载的路径，你下载的路径不动，硬链接的无论怎么改都没关系，应该都不影响。当然前提是没有跨盘操作。

明白了，哎我现在就是想先下载到ssd，然后再转移到hdd，果然有障碍
